src/core/evaluator.js

```diff
@@ -402,6 +403,25 @@ class PartialEvaluator {
     return data;
   }
 
+  loadFontInDocument(name) {
+    const cachedData = this.standardFontDataCache.get(name);
+    if (cachedData) {
+      return cachedData;
+    }
+    const subst = getFontSubstitution(
```

Comment:
Please make sure that this code cannot accidentally run in e.g. Node.js environments:
```suggestion
    if (!this.options.useSystemFonts) {
      return null;
    }
    const subst = getFontSubstitution(
```